auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -297,6 +300,18 @@ public boolean awsAutoScalingEnabled(String accountId) {
         return isEntitlementRegistered(accountId, DATAHUB_AWS_AUTOSCALING);
     }
 
+    public boolean awsImpalaScheduleScalingEnabled(String accountId) {
+        return isEntitlementRegistered(accountId, DATAHUB_AWS_IMPALA_SCHEDULE_BASED_SCALING);
+    }
+
+    public boolean azureImpalaScheduleScalingEnabled(String accountId) {
+        return isEntitlementRegistered(accountId, DATAHUB_AZURE_IMPALA_SCHEDULE_BASED_SCALING);
+    }
+
+    public boolean gcpImpalaScheduleScalingEnabled(String accountId) {
+        return isEntitlementRegistered(accountId, DATAHUB_GCP_IMPALA_SCHEDULE_BASED_SCALING);
+    }
+
```

Comment:
Are these being used? We can remove these.

Reply:
Removed